examples/action/ETH/ETH.xml

```diff
@@ -100,6 +101,20 @@
         </ts:card>
 
         <ts:card type="activity" name="aETHMinted">
+            <ts:origins>
```

Comment:
@colourful-land is the text that goes in here https://app.zeplin.io/project/5d0356be2f25b319dcc5944b/screen/5f101c19f9da398cca6382df defined a label in this block?

Reply:
No, this is the item view